samples/basic/blinky/sample.yaml

```diff
@@ -10,3 +10,31 @@ tests:
     harness: led
     integration_platforms:
       - frdm_k64f
+
+  sample.basic.blinky.egpio:
+    platform_allow: nrf54l15pdk/nrf54l15/cpuapp
+    integration_platforms:
+      - nrf54l15pdk/nrf54l15/cpuapp
+    tags:
+      - LED
+      - gpio
+      - ipc
+    filter: dt_enabled_alias_with_parent_compat("led4", "gpio-leds")
+    depends_on: gpio
+    harness:
```

Comment:
Please change to `harness: console`
Current configuration fails and twister cannot start any testing.